### PR TITLE
[SHELL] Localization of "System Folder" in My Computer types

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- "System folder" in My Computer types is not localized (code indicates "/* FIXME: translate */")

image

## Proposed changes

- Localize this string

image

### CODE REVIEW

File: dll/win32/shell32/folders/CRegFolder.cpp [hunk 1]
```diff
@@ -763,8 +763,8 @@ HRESULT WINAPI CRegFolder::GetDetailsOf(PCUITEMID_CHILD pidl, UINT iColumn, SHEL
             RegCloseKey(hKey);
             return S_OK;
         case 2:        /* type */
-            //return SHSetStrRet(&psd->str, resource_id); /* FIXME: translate */
-            return SHSetStrRet(&psd->str, "System Folder");
+            CString strSysFolder(MAKEINTRESOURCEW(IDS_SYSTEMFOLDER));
+            return SHSetStrRet(&psd->str, strSysFolder);
```
Comment (HBelusca) [inline dll/win32/shell32/folders/CRegFolder.cpp:767]:
```suggestion
            return SHSetStrRet(&psd->str, IDS_SYSTEMFOLDER);
```

Comment (KyleKatarn31) [reply dll/win32/shell32/folders/CRegFolder.cpp]: Done !

File: dll/win32/shell32/lang/pl-PL.rc [hunk 2]
```diff
@@ -1012,4 +1012,6 @@ BEGIN
     IDS_MOVETOTITLE "Zaznacz miejsce, w które chcesz przenieść '%s', a następnie kliknij przycisk Przenieś."
     IDS_MOVEITEMS "Przenoszenie elementów"
     IDS_MOVEBUTTON "Przenieś"
+
+    IDS_SYSTEMFOLDER "System Folder"
```
Comment (Saibamen) [inline dll/win32/shell32/lang/pl-PL.rc:1016]:
```suggestion
    IDS_SYSTEMFOLDER "Folder systemowy"
```

**Review by Saibamen — CHANGES_REQUESTED** [review]
Polish translation

File: dll/win32/shell32/lang/ru-RU.rc [hunk 3]
```diff
@@ -1013,4 +1013,6 @@ BEGIN
     IDS_MOVETOTITLE "Выберите расположение, куда вы хотите переместить '%s', затем нажмите кнопку 'Переместить'."
     IDS_MOVEITEMS "Перемещение элементов"
     IDS_MOVEBUTTON "Переместить"
+
+    IDS_SYSTEMFOLDER "System Folder"
```
Comment (JoseSanchezz) [inline dll/win32/shell32/lang/ru-RU.rc:1017]:
```suggestion
    IDS_SYSTEMFOLDER "Системная папка"
```

**Comment by KyleKatarn31** [comment]
Thanks @JoseSanchezz  and @Saibamen for your contributions !

**Comment by Saibamen** [comment]
BTW How to "add"/show these System Folders in `My Computer` view in Windows XP? By default I see only drivers.

I am asking, because I'm not sure if polish translation for this string is `Folder` (ang. Folder) or `Katalog` (ang. Catalog) and I need to check this somehow. Plus after this I will be able to see polish translation for `Comments` column (https://github.com/reactos/reactos/pull/3167 which was merged in https://github.com/reactos/reactos/commit/e358d1d2197906669e9d20fbba55d2383a0bfe70).

**Comment by learn-more** [comment]
> BTW How to "add"/show these System Folders in `My Computer` view in Windows XP? By default I see only drivers.
> 
> I am asking, because I'm not sure if polish translation for this string is `Folder` (ang. Folder) or `Katalog` (ang. Catalog) and I need to check this somehow. Plus after this I will be able to see polish translation for `Comments` column (#3167 which was merged in [e358d1d](https://github.com/reactos/reactos/commit/e358d1d2197906669e9d20fbba55d2383a0bfe70)).

Install those shell extensions in Windows xp.
They are specific to ReactOS

**Review by HBelusca — APPROVED** [review]

**Pull request merged**